gitlens/gitlens-plus.md

```diff
@@ -100,16 +108,61 @@ Filter options can be accessed from the filter dropdown. This provides the abili
 
 <img src="/wp-content/uploads/filter-options.png" srcset="/wp-content/uploads/filter-options@2x.png" class="img-bordered img-responsive center">
 
+### Minimap (Experimental)
+
+The Minimap provides an additional dimension to the Commit Graph. You can quickly see the activity of the repository, see the HEAD/upstream, branches (local and remote), and easily jump to them. Select the Toggle Minimap icon in the right corner of the Commit Graph top bar to toggle the Minimap on and off. 
+
+<img src="/wp-content/uploads/gl-minimap.png" class="img-bordered img-responsive center">
+
+Minimap Overview:
+
+- Reads left to right - left is the most recent and the right is older
+- Highlighted Region: Commit Graph area that is in view
+- Green Lines: HEAD and upstream
+- Blue Lines: Tags
+- Short Orange Lines: Remote branches
+- Long Orange Lines: Local branches
+
+<div class='callout callout--basic'>
+    <p>Note: We would love to hear your feedback regarding the Minimap. Please share it on the <a href='https://github.com/gitkraken/vscode-gitlens/discussions/2477#discussion-4807133' target='_blank'>GitHub Discussion board</a>.</p>
+</div>
+
+### Deep Linking
+
+Deep Links are links that can be used to open up a ref within the Commit Graph of GitLens. This provides the ability to easily share specific remote repositories, commits, branches, and tags. To generate a deep link right-click the desired ref, hover over _Share_ and select _Copy Link to \<ref\>_. 
```

Comment:
I'm wondering if instead of "ref" we use "resource". Not super important, but "ref" feels more like internal shorthand.